examples/model_selection/plot_cost_sensitive_learning.py

```diff
@@ -660,14 +660,17 @@ def business_metric(y_true, y_pred, amount):
 #
 # The class :class:`~sklearn.model_selection.FixedThresholdClassifier` allows us to
 # manually set the decision threshold. At prediction time, it behave as the previous
-# tuned model but no search is performed during the fitting process.
+# tuned model but no search is performed during the fitting process. Note that here
+# we use :class:`~sklearn.frozen.FrozenEstimator` to wrap the predictive model to
+# avoid any refitting.
 #
 # Here, we will reuse the decision threshold found in the previous section to create a
 # new model and check that it gives the same results.
+from sklearn.frozen import FrozenEstimator
 from sklearn.model_selection import FixedThresholdClassifier
 
 model_fixed_threshold = FixedThresholdClassifier(
-    estimator=model, threshold=tuned_model.best_threshold_, prefit=True
+    estimator=FrozenEstimator(model), threshold=tuned_model.best_threshold_
 ).fit(data_train, target_train)
```

Comment:
It's a bit weird that `.fit(data_train, target_train)` is still needed in this case (just to set the internal `estimator_` attribute).

Maybe `FixedThresholdClassifier.predict` could be changed to avoid calling `        check_is_fitted(self, "estimator_")` and use:

```python
estimator = getattr(self, "estimator_", self.estimator)
...
```

Then the call to `_get_response_values_binary` would naturally call the `check_is_fitted` of the wrapped estimator to make sure that we raise the excepted exception if the underlying estimator has not been fitted beforehand.

This would allow using it with a `FrozenEstimator` without the useless call to `.fit`.

Reply:
I don't think it's odd. It's expected. Fit doesn't just do fit in so many cases. So I would have kept it as is.

But my last commit now does this (it's slightly bigger than the change you suggest).

Note that this means technically, the user doesn't have to wrap the sub-estimator in a `FrozenEstimator`.